src/main/resources/archetype-resources/__rootArtifactId__-module/src/main/resources/META-INF/resources/css/file.css

```diff
@@ -36,30 +34,17 @@
   font-size:12px;
   line-height:1.5
 }
-.file_box .header .headline .options .btn:hover {
-  color:#4b8fbc;
-  background-color:transparent
-}
+
 .file_box .header .headline .options .caret {
   margin-left:6px
 }
-.file_box .header .headline .options .open>.dropdown-toggle {
-  color:#f39c12;
-  background-color:transparent
-}
+
 .file_box .header .headline .options .open>.dropdown-toggle .btn-default {
   background-color:transparent
 }
-.file_box .header .headline .options .open>.dropdown-toggle:hover {
-  color:#4b8fbc
-}
-.file_box .header .headline .options .option {
-  color:#95a5a6;
-  background-color:transparent
-}
-.file_box .header .headline .options .option:hover {
-  color:white
-}
+
+
+
 .file_box .header .headline .options .option .glyphicon {
```

Comment:
If we have migrated to font-awesome completely - this makes no sense.
Update class name or delete css-entry - Look for more glyphicon css classes